Minecraft: Added shuffled recipe list to en_Minecraft.md (#980)
ArchipelagoMW/Archipelago: worlds/minecraft/docs/en_Minecraft.md

> Recipes are removed from the crafting book and shuffled into the item pool. It can also optionally change which

Reviewer: while we are at it I would also like this reworded 
```suggestion
Some recipes are locked from being able to be crafted and shuffled into the item pool. It can also optionally change which
```

structures appear in each dimension. Crafting recipes are re-learned when they are received from other players as item

> checks, and occasionally when completing your own achievements. See below for what recipes are shuffled.

Reviewer: ```suggestion
checks, and occasionally when completing your own achievements. See below for which recipes are shuffled.
```
I *think* which is proper here grammaticaly seeking. Unless someone with more grammar knowledge knows says otherwise.

Victory is achieved when the player kills the Ender Dragon, enters the portal in The End, and completes the credits
sequence either by skipping it or watching hit play out.

> ## What recipes are shuffled?

Reviewer: I dont like the word shuffled here. I would prefer the word locked.

Reviewer: Also "which" again here.
